pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -361,6 +364,16 @@ func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kube
 				return fmt.Errorf("Error setting promiscuous mode on %s: %v", BridgeName, err)
 			}
 		}
+		if plugin.gateway.To4() == nil {
```

Comment:
If we always set the bridge address as above, then we don't need to re-fetch it here.

But even if we did re-fetch it here, don't bother parsing the ip link output.  Just do:

```
link, err := netlink.LinkByName(BridgeName)
if err != nil {
    return err
}
macAddr = link.Attrs().HardwareAddr
```